Error occurs when trying to execute `mecha new module`

I'm trying to create the project following the README.md, but I encountered an error.

https://github.com/hybridgroup/mechanoid?tab=readme-ov-file#example

```
$  export GOPRIVATE=github.com/hybridgroup

$  mecha new example.com/myproject

$ cd myproject

$ mecha new module -template=ping ping
go list -f {{.ImportPath}}: exit status 1
main.go:11:12: pattern modules/ping.wasm: no matching files found
```

--- a/cmd/mecha/create.go
+++ b/cmd/mecha/create.go
@@ -90,11 +90,11 @@ func createFromTemplate(templ, proj string) error {
 
 func getModuleName() (string, error) {
 	var stdout, stderr bytes.Buffer
-	cmd := exec.Command("go", "list", "-f", "{{.ImportPath}}")
+	cmd := exec.Command("go", "list", "-e", "-f", "{{.ImportPath}}")
 	cmd.Stdout = &stdout
 	cmd.Stderr = &stderr
 	if err := cmd.Run(); err != nil {
-		fmt.Printf("go list -f {{.ImportPath}}: %v\n%s%s", err, stderr.Bytes(), stdout.Bytes())
+		fmt.Printf("go list -e -f {{.ImportPath}}: %v\n%s%s", err, stderr.Bytes(), stdout.Bytes())
 		os.Exit(1)
 
 	}

